draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -421,13 +421,11 @@ element unless a matching Disclosure for the second element is received.
 
 ## Example 1: SD-JWT {#example-1}
 
-This example uses the following object as the set of claims that the Issuer is issuing:
+In this example, the simplest SD-JWT is demonstrated.
```

Comment:
there exists no other SD-JWT that could be considered to be more simple than this? :) 

```suggestion
In this example, a simple SD-JWT is demonstrated.
```

Reply:
demonstrated -> demonstrated

or represented?

Reply:
"demostrated" is not a word and "demonstrated" is what was intended